contrib_bots/lib/spotify.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+from future import standard_library
+standard_library.install_aliases()
+import json, urllib.request, urllib.parse, urllib.error
+class SpotifyHandler(object):
+    '''
+    This plugin will allow users to directly search
+    for song, artists, and albums using the Spotify API.
+    It can be called using `@spotify song/artist/album`
```

Comment:
@cosmicAsymmetry could you ask for the extension through the GCI app and explain why you need it? Mentors won't see this comment by default, since GitHub hides them, so I'd like everyone to know what's happening.